README.md

```diff
@@ -9,6 +9,33 @@ Unfortunately, if you're using the desktop version of Slack, you will not** be a
 
 ** However you can still use this theme using this little hack: https://github.com/laCour/slack-night-mode/issues/73#issuecomment-242707078
 
+#### Instructions for macOS
+- Open `/Applications/Slack.app/Contents/Resources/app.asar.unpacked/src/static/ssb-interop.js`
+- Add this snippet to the bottom of that file
+- Replace `/absolute/path/to` with path to your `black.css`
+- You will have to re-add this everytime Slack update itself.
```

Comment:
Reads better as:
`ou will have to re-add this every time Slack updates itself.`